main.cpp

```diff
@@ -331,8 +332,33 @@ static void cipher_examples(void)
     }
 }
 
+static void fake_set_initial_nvseed(void)
+{
+    /* mbedtls_psa_inject_entropy() depends on both MBEDTLS_ENTROPY_NV_SEED and
+     * MBEDTLS_PSA_HAS_ITS_IO being enabled by the Mbed TLS configuration
+     * system. */
+#if defined(MBEDTLS_ENTROPY_NV_SEED) && defined(MBEDTLS_PSA_HAS_ITS_IO)
+    uint8_t seed[MBEDTLS_ENTROPY_MAX_SEED_SIZE];
+
+    /* Calculate a fake seed for injecting. A real factory application would
+     * inject true entropy for use as the initial NV Seed. */
+    for (size_t i = 0; i < sizeof(seed); ++i) {
+        seed[i] = i;
+    }
+
+    int status =  mbedtls_psa_inject_entropy(seed, sizeof(seed));
```

Comment:
Extra space after `=` sign
```suggestion
    int status = mbedtls_psa_inject_entropy(seed, sizeof(seed));
```

Reply:
Fixed